assets/docs/adityakane2001/models/regnety200mf_classification/1.md

```diff
@@ -0,0 +1,42 @@
+# Module adityakane2001/regnety200mf_classification/1
+
+RegNetY 200MF image classifier pretrained on ImageNet-1k.
+
+<!-- asset-path: https://storage.googleapis.com/ak-regnety-savedmodels/tars/regnety200mf_classification.tar.gz  -->
+<!-- task: image-classification -->
+<!-- network-architecture: regnety -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: false -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
+<!-- colab:    -->
+
+## Overview
+
+This model is the implementation of  RegNetY<sup>[1]</sup>  in TensorFlow 2.5.
+
+## Using this model
+
+This model is a pretrained classifier on ImageNet-1k. This means that you can directly classify images over 1000 predefined classes.
+
+```python
+model = tf.keras.Sequential([
+    hub.KerasLayer("https://tfhub.dev/adityakane2001/regnety200mf_classification/1")
+])
+predictions = model.predict(images) 
+```
+
+Inputs to the model must:
+1. be four dimensional Tensors of the shape `(batch_size, height, width, num_channels)`. Note that the model expects images with  `channels_last`  property. `height`,`width` and `batch_size` can be any value based on user's preference.
+2. have pixel values in the range `[0, 255]`.
```

Comment:
We later moved normalization part to inside the model because of performance issues. IMO this provides with an added advantage, for direct classification one doesn't have to resize the images to (224, 224). But please tell me if you think it's better to remove that part. For reference, please see [PreStem](https://github.com/AdityaKane2001/regnety/blob/4e41a3c56a623d6c6fca1dff0e425c5143cbfe6b/regnety/models/blocks.py#L23-L55) class.

Reply:
Ah that's a shame, no worries. Agreed that accepting dynamic image sizes is simpler for the user. It might still be worth mentioning that they are resized to 224x224 internally so that users don't use unnecessarily large images.